Echo.Process/SessionId.cs

```diff
@@ -11,7 +11,7 @@ namespace Echo
     /// <remarks>
     /// It enforces the rules for session IDs.
     /// </remarks>
-    public struct SessionId : IEquatable<SessionId>, IComparable<SessionId>, IComparable
+    public class SessionId : Record<SessionId>
```

Comment:
This is bad, please put it back to being a `struct`, this will cause unnecessary allocations all over the place.